flink-core/src/main/java/org/apache/flink/api/common/io/GenericCsvInputFormat.java

```diff
@@ -445,7 +445,12 @@ protected int skipFields(byte[] bytes, int startPos, int limit, byte[] delim) {
 			// quoted string parsing enabled and field is quoted
 			// search for ending quote character
 			i++;
-			while(i < limit && bytes[i] != quoteCharacter) {
+
```

Comment:
Would this be the same as `while (i < limit && (bytes[i] != quoteCharacter || bytes[i-1] == 92))`?